projects/table-builder/src/lib/components/generic-table/generic-table.component.ts

```diff
@@ -101,7 +101,7 @@ export class GenericTableComponent implements OnInit {
     );
     this.dataSource.sort = this.sort;
     this.dataSource.sortData = (data: {}[], sort: MultiSortDirective) =>
-      orderBy(data, sort.rules.map(r => r.active), sort.rules.map(r => r.direction as direc ));
+      orderBy(data, sort.rules?.map(r => r.active), sort.rules?.map(r => r.direction as direc ));
```

Comment:
How can there not be rules?

Reply:
It was done due to a failing test.
I found a better way to fix the test.